docs/collections.rst

```diff
@@ -0,0 +1,326 @@
+| ``full_shap``          | XGBoost               | A matrix of (nsmaple, |
+|                        |                       | nfeatures + 1) with   |
+|                        |                       | each record           |
+|                        |                       | indicating the        |
+|                        |                       | feature contributions |
+|                        |                       | (`SHAP                |
+|                        |                       | valu                  |
+|                        |                       | es <https://github.co |
+|                        |                       | m/slundberg/shap>`__) |
+|                        |                       | for that prediction.  |
+|                        |                       | Computed on training  |
+|                        |                       | data with             |
+|                        |                       | `predic               |
+|                        |                       | t() <https://github.c |
+|                        |                       | om/slundberg/shap>`__ |
++------------------------+-----------------------+-----------------------+
+| ``average_shap``       | XGBoost               | The sum of SHAP value |
+|                        |                       | magnitudes over all   |
+|                        |                       | samples. Represents   |
+|                        |                       | the impact each       |
+|                        |                       | feature has on the    |
+|                        |                       | model output.         |
++------------------------+-----------------------+-----------------------+
+| ``trees``              | XGBoost               | Boosted tree model    |
+|                        |                       | given by              |
+|                        |                       | `trees_to_dataframe(  |
+|                        |                       | ) <https://xgboost.re |
+|                        |                       | adthedocs.io/en/lates |
+|                        |                       | t/python/python_api.h |
+|                        |                       | tml#xgboost.Booster.t |
+|                        |                       | rees_to_dataframe>`__ |
++------------------------+-----------------------+-----------------------+
+
+Default collections saved
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The following collections are saved regardless of the hook
+configuration.
+
+============== ===========================
+Framework      Default collections saved
+============== ===========================
+``TensorFlow`` METRICS, LOSSES, SM_METRICS
+``PyTorch``    LOSSES
+``MXNet``      LOSSES
+``XGBoost``    METRICS
+============== ===========================
+
+If for some reason, you want to disable the saving of these collections,
```

Comment:
We should tell customers to set `debugger_hook_config=False` in the estimator, this is a simpler alternative.